features/bootstrap/DatasetViewContext.php

```diff
@@ -416,7 +416,13 @@ public function iHaveSetPagesizeToOn($arg1, $arg2)
         }
     }
 
-
-
+    /**
+     * @Then there is a :arg1 meta tag :arg2 with value :arg3
+     */
+    public function thereIsAMetaTagWithValue($arg1, $arg2, $arg3)
```

Comment:
PR #516 has functions with similar names: `thereIsAMetaTagWithValue($arg1, $arg2)` and `thereIsMetaTagWithValues($arg1, $arg2, $arg3)` which will conflict with the above function. I think its worth having a think about how to name these functions that do similar things.